media/css/firefox/os/firefox-os-new.less

```diff
@@ -0,0 +1,960 @@
+@import "../../sandstone/lib.less";
+
+* {
+    .border-box();
```

Comment:
This has impacted the family nav, leaving space between the border and subnav.
![screen shot 2015-02-27 at 1 47 47 pm](https://cloud.githubusercontent.com/assets/317532/6419936/46b81d12-be87-11e4-8933-aa4423872442.png)
Easy fix may be to just apply `border-box` to everything inside `#wrapper`?